src/Raven.Client/Documents/Operations/Backups/StartBackupOperation.cs

```diff
@@ -32,16 +32,18 @@ internal sealed class StartBackupCommand : RavenCommand<OperationIdResult<StartB
             private readonly long _taskId;
             private readonly long? _operationId;
             private readonly DateTime? _startTime;
+            private readonly bool _inProgressOnAnotherShard;
```

Comment:
this seems to be relevant only for sharded databases, wonder if we can (and how easy would that be) to go with inheritance here..
e.g. to have  `StartShardedBackupOperation : StartBackupOperation` which is not relevant for the client API, but will be used only from server.

I guess more classes like that would need to be introduced.. like handlers?

Reply:
Not relevant, because @lastav5 will change the approach here and open a new PR